docs: add info about env var usage (#10755)
apache/apisix: docs/en/latest/profile.md

Initialize and start APISIX in standalone mode, requests to `/anything` should now be forwarded to `httpbin.org:80/anything`.

> *WARNING*: When using docker to deploy in standalone mode. New environment variables added in `apisix.yaml` while APISIX is running will only take effect after a reload.

Reviewer: ```suggestion
*WARNING*: When using docker to deploy in standalone mode. New environment variables added to `apisix.yaml` while APISIX has been initialized will only take effect after a reload.
```
